types/bull/index.d.ts

```diff
@@ -141,13 +148,30 @@ declare namespace Bull {
      */
     retry(): Promise<void>;
 
+    /**
+     * Ensure this job is never ran again even if attemptsMade is less than job.attempts.
+     */
+    discard(): Promise<void>;
+
     /**
      * Returns a promise that resolves to the returned data when the job has been finished.
      * TODO: Add a watchdog to check if the job has finished periodically.
      * since pubsub does not give any guarantees.
      */
     finished(): Promise<any>;
 
+    /**
+     * Moves a job to the `completed` queue. Pulls a job from 'waiting' to 'active'
+     * and returns a tuple containing the next jobs data and id. If no job is in the `waiting` queue, returns null.
+     */
+    moveToCompleted(returnValue?: string, ignoreLock?: boolean): Promise<string[] | null>;
+
+    /**
+     * Moves a job to the `failed` queue. Pulls a job from 'waiting' to 'active'
+     * and returns a tuple containing the next jobs data and id. If no job is in the `waiting` queue, returns null.
+     */
+    moveToFailed(errorInfo: { message: string; }, ignoreLock?: boolean): Promise<void>;
```

Comment:
you could add the returning tuple to the return type here

Reply:
Done, changed it to `[any, JobId] | null` as described in docs

Reply:
Done, changed it to `[any, JobId] | null` as described in docs